tests/carry/golden.model.xml

```diff
@@ -1,3 +1,3 @@
 <models xmlns:xi="http://www.w3.org/2001/XInclude">
-  <xi:include href="cblock/cblock.model.xml" xpointer="xpointer(models/child::node())"/>
+  <xi:include href="cblock/golden.model.xml" xpointer="xpointer(models/child::node())"/>
```

Comment:
`golden` files should never be included.

Reply:
I fixed all includes so they don't have `golden` files